lib/jenkins_pipeline/app.rb

```diff
@@ -2,6 +2,7 @@ module JenkinsPipeline
   class App < Sinatra::Base
 
     set :public_folder, './public'
+    set :bind, "0.0.0.0"
```

Comment:
Since we are running Sinatra from a *kind of VM* we have to redirect some ports to outside world.
If we try to run a Sinatra app inside a Vagrant box we will have the same behaviour.